src/modules/init.js

```diff
@@ -0,0 +1,26 @@
+import refresh from './leaderboardAPI/refresh.js';
+import postNewScore from './leaderboardAPI/postNewScore.js';
+
+export default (newLB, gameID) => {
+  // Listen to Refresh button
+  const refreshBtn = document.querySelector('#refresh');
+  refreshBtn.addEventListener('click', () => {
+    refresh(newLB, gameID);
+  });
+
+  // Listen to the submit button
+  const submitBtn = document.querySelector('#submit');
+  submitBtn.addEventListener('click', (e) => {
+    e.preventDefault();
+    const newName = document.querySelector('#new-name');
+    const newScore = document.querySelector('#new-score');
+    if (newName.value !== '' && newScore.value !== '') {
+      postNewScore(gameID, newName.value, newScore.value);
+      newName.value = '';
+      newScore.value = '';
+    } else {
+      // eslint-disable-next-line no-alert
```

Comment:
[ ] [REQUIRED] Good job so far! Please fix linter errors without this kind of comment `/* eslint-disable ... */` because this type of comment is used in very few cases, and in your case, you can solve such errors yourself, and I think if implement my suggestion that this will increase your work experience and skills 😃